lib/mpl_toolkits/axisartist/axisline_style.py

```diff
@@ -140,6 +155,27 @@ def new_line(self, axis_artist, transform):
     _style_list["->"] = SimpleArrow
 
     class FilledArrow(SimpleArrow):
+        def __init__(self, size=1, facecolor=mpl.rcParams['axes.edgecolor']):
```

Comment:
Seems odd to set to an rcParam in a definition line... 
I believe that would be set at import time, so if the rcParams dict changed after import it would still have the old value for the `__init__` default.

Reply:
A (quick) scan of some other places which have defaults set to rcParams shows the more standard pattern is to have the default set to `None` and then if it is None get the rcParam at call time.

Reply:
The lookup should be done late (via the `None` dance).  

This will set the default to be what ever the default was at class definition time which I _think_ only is done once with this class (even though it is nested).

Reply:
Yeah, you are both correct...

Fixed!